integration/spark-common/src/main/scala/org/apache/spark/rdd/CarbonMergeFilesRDD.scala

```diff
@@ -82,19 +84,33 @@ object CarbonMergeFilesRDD {
             readFileFooterFromCarbonDataFile).collect()
         }
       } catch {
-        case _: Exception =>
-          if (CarbonCommonConstants.CARBON_MERGE_INDEX_IN_SEGMENT_DEFAULT.toBoolean) {
-            new CarbonMergeFilesRDD(
-              sparkSession,
-              carbonTable,
-              segmentIds,
-              segmentFileNameToSegmentIdMap,
-              carbonTable.isHivePartitionTable,
-              readFileFooterFromCarbonDataFile).collect()
-          }
+        case ex: Exception =>
+          val message = "Merge Index files request is failed " +
```

Comment:
Better add a property to let the user decides whether merge index job fail should fail the whole job or just log it and ignore it.  Usually, for long loads it is not good to fail the job just for merge index file.

Reply:
done